bootstrap/terraform/aws-bootstrap/main.tf

```diff
@@ -3,6 +3,8 @@ data "aws_availability_zones" "available" {}
 data "aws_caller_identity" "current" {}
 
 module "vpc" {
+  count =  var.create_cluster ? 1:0
```

Comment:
similar there's a create vpc field we can use to turn this module off

Reply:
I don't think that variable was ever implemented. If not, it's easier to consolidate them.